storage/innobase/xtrabackup/test/t/xb_export.sh

```diff
@@ -136,59 +144,46 @@ vlog "Re-initializing database server"
 stop_server
 rm -rf ${MYSQLD_DATADIR}
 start_server $mysql_extra_args
-init_schema incremental_sample test $max_indices_count "n"
+init_schema incremental_sample test1 $max_indices_count "n"
 init_schema incremental_sample test2 $indices_count "n"
 init_schema incremental_sample test3 $indices_count "y"
-# check if we can import all the columns type in mysql
-mysql -e 'create table test4(col1 bool, col2 int, col3 float, col4 double,
-col5 timestamp, col6 long, col7 date, col8 time, col9 datetime, col10 year,
-col11 varchar(20), col12 bit ,col13 decimal, col14 blob, col16 json,
-col17 mediumtext, col18 enum("01","2"), col19 SET("0","1","2"),
-col20 varchar(255) character set latin1 )' incremental_sample;
-mysql -e 'CREATE TABLE test5(a int, b int, PRIMARY KEY (a), KEY (b DESC))' incremental_sample
-mysql -e 'CREATE TABLE test6(a int)' incremental_sample
-mysql -e 'ALTER TABLE test6 ADD COLUMN v1 VARCHAR(255), ALGORITHM=INSTANT' incremental_sample
-mysql -e 'CREATE TABLE test7(a int) row_format=compressed' incremental_sample
-mysql -e 'CREATE TABLE test8(c1 INT) ENGINE = InnoDB' incremental_sample
-mysql -e 'ALTER TABLE test8 ADD COLUMN c2 INT DEFAULT 500' incremental_sample
-vlog "Database was re-initialized"
 
-mysql -e "alter table test discard tablespace;" incremental_sample
-mysql -e "alter table test2 discard tablespace;" incremental_sample
-mysql -e "alter table test3 discard tablespace;" incremental_sample
-mysql -e "alter table test4 discard tablespace;" incremental_sample
-mysql -e "alter table test5 discard tablespace;" incremental_sample
-mysql -e "alter table test6 discard tablespace;" incremental_sample
-mysql -e "alter table test7 discard tablespace;" incremental_sample
-mysql -e "alter table test8 discard tablespace;" incremental_sample
+#create default tables
+create_tables
+
+vlog "Database was re-initialized"
+for ((c=1; c<=$table_count; ++c ))
+do
+  mysql -e "alter table test$c discard tablespace;" incremental_sample
+done
 
 xtrabackup --datadir=$mysql_datadir --prepare --export \
     --target-dir=$backup_dir
 
 ls -tlr $backup_dir/incremental_sample/*cfg
 cfg_count=`find $backup_dir/incremental_sample -name '*.cfg' | wc -l`
-vlog "Verifying .cfg files in backup, expecting 10."
-if [ $cfg_count -ne 10 ]
+vlog "Verifying .cfg files in backup, expecting 11."
```

Comment:
are we adding a new config on export ?

Reply:
we were exporting 10 cfg now we would be exporting 11 . new config added as test9